crates/currency_conversion/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "currency_conversion"
+description = "Currency conversion for cost based routing"
+version = "0.1.0"
+edition.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[dependencies]
+# First party crates
+api_models = { version = "0.1.0", path = "../api_models", package = "api_models" }
```

Comment:
I think `common_enums` should suffice for this crate. Use `common_enums` instead of `api_models`